Browser/Fable.Helpers.WebWorker.fs

```diff
@@ -0,0 +1,61 @@
+module Fable.Helpers.WebWorker
```

Comment:
This files should be a standalone library.

`Fable.Import.Browser` package is only a binding and do not include the sources files. So the helpers can't be compiled.

I think having `Fable.Import.Browser` with both:
- Browser API
- WebWorker API

is fine. If people want a better UX then they will be able to add `Fable.Helpers.WebWorker` library.